binding.gyp

```diff
@@ -139,156 +139,6 @@
         'CLANG_CXX_LANGUAGE_STANDARD':'c++14',
         'GCC_VERSION': 'com.apple.compilers.llvm.clang.1_0'
       }
```

Comment:
This is a **temporary removal**, since I was getting other boost test-related build errors.